lib/workers/pr/index.ts

```diff
@@ -50,10 +58,9 @@ export async function addAssigneesReviewers(
   if (config.assigneesFromCodeOwners) {
     assignees = await addCodeOwners(assignees, pr);
   }
-  assignees = await filterUnavailableUsers(config, assignees);
   if (assignees.length > 0) {
     try {
-      assignees = assignees.map(noLeadingAtSymbol);
```

Comment:
Tiny behavior change: We now remove duplicate assignees (as we already did for reviewers). I cannot imagine a situation where this would cause problems, happy to have another brain think about this 🧐